server/plugin/command/find_meeting_times.go

```diff
@@ -4,28 +4,37 @@
 package command
 
 import (
-	"encoding/json"
-	"fmt"
+	"strings"
 
 	"github.com/mattermost/mattermost-plugin-msoffice/server/remote"
 	"github.com/mattermost/mattermost-plugin-msoffice/server/utils"
 )
 
 func (c *Command) findMeetings(parameters ...string) (string, error) {
 	meetingParams := &remote.FindMeetingTimesParameters{}
-	// meetings, err := c.API.FindMeetingTimes(time.Now(), time.Now().Add(14*24*time.Hour))
+
```

Comment:
1/5 We should support entering users as `@username`, or as a `myaddress@email.sucks`, see above comment for an example.

1/5 recommend a consistent syntax for providing a list of users, 0/5 suggest `--users @user1,@user2`. Using a space-separated list (i.e. the tail of the command line) is appealing, but then it is also appealing to use for other purposes, like typing in a subject. Ultimately, we may chose something in between consistency and practicality, e.g. going for `/... invite @user1 @user2 2020-01-01 at 12:00AM Celebrate New Year's ` and smart parsing based on the values themselves.

Reply:
> Ultimately, we may chose something in between consistency and practicality, e.g. going for /... invite @user1 @user2 2020-01-01 at 12:00AM Celebrate New Year's and smart parsing based on the values themselves.

I think this is very tedious UX. I certainly would not want to manually type out dates or times. Ideally we would have an as-simple-as-possible equivalent in a UI form. I think it would be about 8 mana to implement, without username autocomplete for the attendees. Everything else would be entered by value as far as I can tell. Datetime components are pretty straightforward to configure.